v2018.2.x: bump OpenWrt and other feeds and add v2018.2.4 release notes (#1896)
freifunk-gluon/gluon: docs/releases/v2018.2.4.rst

Gluon 2018.2.4
==============

Bugfixes
~~~~~~~~

* Fixes device alias for Ubiquiti UniFi AC LR. (`#1834 <https://github.com/freifunk-gluon/gluon/issues/1834>`_)

* Uses correct ath10k firmware for OCEDO Koala. (`#1838 <https://github.com/freifunk-gluon/gluon/pull/1838>`_)

> * Fixes some batman-adv bugs with backports from 2019.4 and 2019.5 by updating the openwrt-routing packages feed

Reviewer: `some` -> `various`

Author: fixed

Other changes
~~~~~~~~~~~~~

* Linux kernel has been updated to either
  

>   - 4.9.207 (ar71xx, brcm2708, mpc85xx) or

Reviewer: Would drop the "or" in this listing

Author: was copied from v2018.2.3 and is necessary because of the "either" in the sentence above, isn't it?

Reviewer: I would drop the either also then. But this is more of a personal preference.

Gluon 2018.2.4
==============

> Important Note

Reviewer 2: ```suggestion
End of life
```

~~~~~~~~~~~~~~

> This release will be the last one of the Gluon branch v2018.2.x , an update to v2019.1.x is strongly recommended and should be fairly easy.

Reviewer 2: ```suggestion
This will be the final release of the v2018.2.x series. Updating to the v2019.1.x release series is the recommended course of action, which should be fairly easy.
```

Bugfixes
~~~~~~~~

> * Fixes device alias for Ubiquiti UniFi AC LR. (`#1834 <https://github.com/freifunk-gluon/gluon/issues/1834>`_)

Reviewer 2: ```suggestion
* Fixes device alias for Ubiquiti UniFi AC LR. (`#1834 <https://github.com/freifunk-gluon/gluon/issues/1834>`_)
  Autoupdates on this model were impossible before, since we were missing the proper device alias.
```

> * Uses correct ath10k firmware for OCEDO Koala. (`#1838 <https://github.com/freifunk-gluon/gluon/pull/1838>`_)

Reviewer 2: ```suggestion
* Add correct ath10k firmware package for OCEDO Koala. (`#1838 <https://github.com/freifunk-gluon/gluon/pull/1838>`_)
```
